frontend/src/topic/ApiDocs/views/index.js

```diff
@@ -0,0 +1,197 @@
+import CSSModules from 'react-css-modules';
+import Helmet from 'react-helmet';
+import React from 'react';
+
+import styles from './styles.scss';
+import { pageTitles } from '../../../common/utils/labels';
+import { RestBuilder } from '../../../public/utils/rest';
+import {
+    urlForApiDocs,
+} from '../../../common/rest';
+
+
+// Todo move this to common utils module
+const isObjectEmpty = obj => (
+    Object.keys(obj).length === 0 && obj.constructor === Object
+);
+
+
+@CSSModules(styles, { allowMultiple: true })
+export default class ApiDocs extends React.PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            docs: {},
+            expanded: [],
+        };
+    }
+
+    componentWillMount() {
+        this.fetchApiDocs(urlForApiDocs);
+    }
+
+    fetchApiDocs = (url) => {
+        // Stop any retry action
+        if (this.apiDocsRequest) {
+            this.apiDocsRequest.stop();
+        }
```

Comment:
1. apiDocsRequest must also be closed using method close() in componentWillUnmount
2. Properties must be set for navbar in componentWillMount.

Reply:
Not sure if it's intentional or fixed already but Login view has not implemented `componentWillUnmount` as well. I missed it because I was copying what it did.